💄 Logging for image size

aileen opened the pull request:
no issue

- added debug logs to image size util and related fn:
    - when fetched via network request
    - when fetched from storage
    - when added to cache
    - when read from cache

kirrg001 commented on core/server/adapters/storage/utils.js:
```diff
@@ -1,27 +1,47 @@
+ * @returns {Boolean}
+ */
+
+exports.isLocalImage = function isLocalImage(imagePath) {
```
This function is basically doing what `getLocalFileStoragePath` is doing, except of the return value 🤔 
As these conditions are quite long and it takes a bit to read them, how about doing

```
x = getLocalFileStoragePath(path)
if (x !== path) {
   return true;
}

return false;
```

AileenCGN replied:
Thank you for this! Totally makes sense to me and looks sooo much better now!

kirrg001 commented:
FYI: An empty file was added -> `core/test/unit/utils/request_spec.js`.

AileenCGN commented:
@kirrg001 Please don't merge this yet. I'm doing further investigations and will update this PR in a bit.

aileen committed:
📖  Logging for image size

no issue

- added debug logs to image size util and related fn:
    - when fetched via network request
    - when fetched from storage
    - when added to cache
    - when read from cache

AileenCGN commented:
@ErisDS This is ready to go!
